moveit_ros/moveit_servo/include/moveit_servo/servo_calcs.h

```diff
@@ -278,7 +278,7 @@ class ServoCalcs
   // internal_joint_state_ is used in servo calculations. It shouldn't be relied on to be accurate.
   // original_joint_state_ is the same as incoming_joint_state_ except it only contains the joints the servo node acts
   // on.
-  sensor_msgs::msg::JointState internal_joint_state_, original_joint_state_;
+  sensor_msgs::msg::JointState internal_joint_state_, original_joint_state_, last_joint_state_;
```

Comment:
Let's add a comment above this. Here's my suggestion, feel free to modify it.
```suggestion
  // last_joint_state_ caches the previous `original_joint_state_`
  sensor_msgs::msg::JointState internal_joint_state_, original_joint_state_, last_joint_state_;
```

Reply:
Thanks . I forgot to add this.